product.html

```diff
@@ -0,0 +1,31 @@
+<tr>
```

Comment:
Kindly refresh my memory on why we need this file or was it just part of ensuring we modularized the code?

Reply:
Just modularizing the code!